packages/ses/src/error/assert.js

```diff
@@ -54,6 +54,50 @@ freeze(quote);
  */
 const hiddenDetailsMap = new WeakMap();
 
+/**
+ * @param {HiddenDetails} hiddenDetails
+ * @returns {string}
+ */
+const getMessageString = ({ template, args }) => {
```

Comment:
verifying that moving this function didn't change it is a bit tedious, but I think I convinced myself that there is no change.

If it there was more code to compare, I would suggest a separate "no observable behavioral change" refactor commit.

Reply:
I like that. Thanks.